docs/CI-CD/ACR.md

```diff
@@ -0,0 +1,90 @@
+# Setup CI-CD with ACR
+
+> Setup Github Actions to build and push the helium container to Azure Container Registry (ACR)
+
+## Enable GitHub Actions
+
+By default, GitHub Actions are disabled for forked repositories. To enable:
+
+- Go to your forked repository on github.com
+- Click on the Actions tab
+- Click on the enable button
+
+## Create Service Principal for ACR
+
+> Note: These instructions depend on the setup environment variables being set correctly
+
+```bash
+
+# source environment variables if needed
+source ~/.azure/.helium.env
+
+```
+
```

Comment:
In the main Helium README.md - it would help to have a link to SP and secret setup - maybe around like 190.

Reply:
There is another PR open with major changes to the readme, so can't make changes until that is merged.